wizeline/wize-q-remix
# feat(migration): migrate questionDetail component (retrive data)

#### What does this PR do?
- Initial migration for the QuestionDetail Component (just to show the information)

#### How should this be manually tested?
(List of steps to reproduce, corroborate or tests to run. Write this section clear enough so that external users can also follow it and test the fix.)
1. Login
2. Click in any question
3. Verify that you can see:
         - The question,
         - The number of likes and comments 
         -  Pin question (if the question has the attribute)
         -  Who create the question
         - The department.
4. Click on the `Back` button to verify its functionality
5. 
#### What are the relevant tickets?
Closes #5 

#### Screenshots
<img width="1790" alt="Screen Shot 2022-10-12 at 19 03 10" src="https://user-images.githubusercontent.com/97203617/195469373-b155694d-9e3d-4117-adc7-7d42c4dc8d33.png">

👀 Until this point, we can't see the comments or do all the actions related to this page because we don't have all the controller for each action.
